lmnet/lmnet/layers/layers.py

```diff
@@ -50,7 +50,6 @@ def conv2d(
     strides=1,
     padding="SAME",
     activation=tf.nn.relu,
-    kernel_initializer=tf.contrib.layers.xavier_initializer(),
```

Comment:
@ananno Sorry, I have one question, before your change, we use `layers.xavier_initializer` and ignore `variance_scaling_initializer` by bug. 
Your fix change initializer `xavier_initializer` to `variance_scaling_initializer` and it's affect training accuracy?  🤔

Reply:
I didn't trained with both. But theoretically it would better perform with `variance_scaling_initializer` as it works better with `ReLU` activation whereas `xavier_initializer` works better with sigmoid activation.
Ref. Xavier : http://proceedings.mlr.press/v9/glorot10a/glorot10a.pdf
Ref. Variance Scaling : https://arxiv.org/pdf/1502.01852v1.pdf

Reply:
I want @tkng san's opinion about this.

Reply:
Moreover, before we didn't pass `kernel_initializer` into the `tf.layers.conv2d` function anyway, so it would use the default value always and our selection through config was not being applied.

Reply:
I see.